src/main/java/frc/robot/Subsystems/Drivetrain.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.revrobotics.ColorSensorV3;
+import com.spikes2212.command.drivetrains.TankDrivetrain;
+import edu.wpi.first.wpilibj.ADXRS450_Gyro;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import frc.robot.RobotMap;
+
+public class Drivetrain extends TankDrivetrain {
+    private static Drivetrain drivetrain;
+
+    private ADXRS450_Gyro gyro;
+    private Encoder leftEncoder, rightEncoder;
+    private ColorSensorV3 colorSensor1, colorSensor2;
```

Comment:
wrong names

Reply:
What are the right names

Reply:
rightColorSensor and leftColorSensor?
index things **only** when there is no other option to distinguish between them